packages/itwinui-react/src/core/Buttons/SplitButton/SplitButton.tsx

```diff
@@ -71,14 +79,17 @@ export const SplitButton = React.forwardRef((props, forwardedRef) => {
 
   return (
     <Box
-      className={cx(className, 'iui-button-split', {
-        'iui-disabled': props.disabled,
-      })}
-      style={style}
-      title={title}
-      ref={ref}
```

Comment:
this `ref` is being used for calculating the menu width so needs to be brought back. you can rename it `wrapperRef` to prevent confusion

Reply:
you need to rename the existing variable, not create a new one

https://github.com/iTwin/iTwinUI/assets/9084735/826d2555-fc2c-4be3-956d-0ec3219c8c75

Reply:
Thanks. Updated